frontend/src/features/admin-form/responses/AdminSubmissionsService.ts

```diff
@@ -18,9 +18,13 @@ export const countFormSubmissions = async ({
   dates,
 }: {
   formId: string
-  dates?: SubmissionCountQueryDto
+  dates?: Partial<SubmissionCountQueryDto>
 }): Promise<number> => {
   const queryUrl = `${ADMIN_FORM_ENDPOINT}/${formId}/submissions/count`
+  if (dates && (!dates.startDate || !dates.endDate)) {
+    delete dates.startDate
+    delete dates.endDate
```

Comment:
Why is this necessary; what happens if we simply pass in undefined startDate and endDate? 

Also should we allow a case where only the startDate or endDate is specified? Right now both must be specified to pass the test